semgrep-core/tests/OTHER/rules/metavar_cond.yaml

```diff
@@ -5,6 +5,8 @@ rules:
   match:
     and:
     - foo($ARG)
```

Comment:
But then isn't it weird that you have a 'and' with just one element in it?

Reply:
This doesn't need to be a singular `and`. We could just use a `pattern`. I dunno why it's like that here.